spec/services/articles/feed_spec.rb

```diff
@@ -180,302 +180,6 @@
-      let(:user) { create(:user, experience_level: 9) }
-
-      it "returns negative of (absolute value of the difference between article and user experience) divided by 2" do
-        expect(feed.score_experience_level(article)).to eq(-1)
-      end
-    end
-
-    context "when the user does not have an experience level set" do
-      let(:user) { create(:user, experience_level: nil) }
-
-      it "uses a value of 5 for user experience level" do
-        expect(feed.score_experience_level(article)).to eq(-1)
-      end
-    end
-  end
-
-  describe "#score_comments" do
-    let(:article_with_one_comment) { create(:article) }
-    let(:article_with_five_comments) { create(:article) }
-
-    before do
-      create(:comment, user: user, commentable: article_with_one_comment)
-      create_list(:comment, 5, user: user, commentable: article_with_five_comments)
-      article_with_one_comment.update_score
-      article_with_five_comments.update_score
-      article_with_one_comment.reload
-      article_with_five_comments.reload
-    end
-
-    context "when comment_weight is default of 0" do
-      it "returns 0 for uncommented articles" do
-        expect(feed.score_comments(article)).to eq(0)
-      end
-
-      it "returns 0 for articles with comments" do
-        expect(article_with_five_comments.comments_count).to eq(5)
-        expect(feed.score_comments(article_with_five_comments)).to eq(0)
-      end
-    end
-
-    context "when comment_weight is higher than 0" do
-      before { feed.instance_variable_set(:@comment_weight, 2) }
-
-      it "returns 0 for uncommented articles" do
-        expect(feed.score_comments(article)).to eq(0)
-      end
-
-      it "returns a non-zero score for commented upon articles" do
-        expect(feed.score_comments(article_with_one_comment)).to be > 0
-      end
-
-      it "scores article with more comments high than others" do
-        expect(feed.score_comments(article_with_five_comments)).to be > feed.score_comments(article_with_one_comment)
-      end
-    end
-  end
-
-  describe "#rank_and_sort_articles" do
-    let(:article1) { create(:article) }
-    let(:article2) { create(:article) }
-    let(:article3) { create(:article) }
-    let(:articles) { [article1, article2, article3] }
-
-    context "when number of articles specified" do
-      let(:feed) { described_class.new(number_of_articles: 1) }
-
-      it "only returns the requested number of articles" do
-        expect(feed.rank_and_sort_articles(articles).size).to eq 1
-      end
-    end
-
-    it "returns articles in scored order" do
-      allow(feed).to receive(:score_single_article).with(article1).and_return(1)
-      allow(feed).to receive(:score_single_article).with(article2).and_return(2)
-      allow(feed).to receive(:score_single_article).with(article3).and_return(3)
-
-      expect(feed.rank_and_sort_articles(articles)).to eq [article3, article2, article1]
-    end
-  end
-
-  describe ".globally_hot_articles" do
```

Comment:
These tests are missing from the refactor: they cover important edge cases and should be included (they may need to become integration level tests now that these classes have been split up

Reply:
How to turn them integration level tests? I removed this test `.globally_hot_articles` because I turned it a private method because it's only used by one method inside the class.

Reply:
@joshpuetz any thoughts?